File: public/phase-2-20160620022203/C/mssum1.c
// Name : MSSUM1
#include <stdio.h>
int main() {
	// input:
	// n :  the number of elements in the array
	// q :  the number of queries correspondingly
	// a :  the array elements
	// lr : each row corresponds to l and r
	int n, q, a[100000], lr[100000][2], sum_of_replies=25, index, idx, jdx;
	char lr_tmp[2];
	scanf("%d %d", &n, &q);
	for(index = 0; index< n; index++) {
		scanf("%d", &a[index]);
	}
	for(idx = 0; idx< q; idx++) {
		scanf("%s", &lr_tmp[0]);
		for(jdx = 0;jdx<2;jdx++) {
			lr[idx][jdx] = lr_tmp[jdx];
		}
	}


	// write your code here
	// store your results in `sum_of_replies`

	// output
	printf("%d", sum_of_replies);
	return 0;
}

File: public/phase-2-20160620022203/PYTHON/mssum1.py
#!/usr/bin/python
# Name : MSSUM1
# input:
# n :  the number of elements in the array
# q :  the number of queries correspondingly
# a :  the array elements
# lr : each row corresponds to l and r
elements = raw_input().strip().split(' ')
n = int(elements[0])
q = int(elements[1])
a_elements = raw_input().strip().split(' ')
a = map(int, a_elements)
lr = []
for _ in xrange(q):
	lr_elements = raw_input().strip().split(' ')
	lr.append(map(int, lr_elements))


# write your code here
# store your results in `sum_of_replies`

# output

# Dummy Data
sum_of_replies = 25

print(sum_of_replies)
